safe-global/web-core
# [EIP-1271] fix: remove explorer link from prepare signature

## What it solves

Resolves #1395

## How this PR fixes it

The explorer link has been removed from the `EthHashInfo` of the `preparedSignature`.

## How to test it

Open a message that has been fully confirmed and observe no explorer link next to the prepared signature.

## Screenshots

![image](https://user-images.githubusercontent.com/20442784/208639088-b128e2dd-2e1f-4a98-81e1-a3967e21c077.png)
